controllers/topic_controller_test.go

```diff
@@ -0,0 +1,63 @@
+package controllers
+
+import (
+	"testing"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestConvertSecret(t *testing.T) {
+	testSecret := secretData{
```

Comment:
and then `secretData` must be made public and gotten from the `controllers` module.